package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apttus.ui.ecom.rc",
-  "version": "2303.0.0",
+  "version": "2402.0.0",
```

Comment:
Version shouldn't be updated here.

Reply:
Doubt: since our sdk versions are updated to 2402.X.X shouldn't this version also be in sync with the others?

Reply:
Since this is upgrade from october'23, base package version shouldn't be changed.